dags/openshift_nightlies/scripts/run_benchmark.sh

```diff
@@ -14,7 +14,7 @@ done
 setup(){
     mkdir /home/airflow/workspace
     cd /home/airflow/workspace
-    git clone https://github.com/cloud-bulldozer/e2e-benchmarking
+    git clone -b fix-run_comapre-dot-sh https://github.com/kedark3/e2e-benchmarking
```

Comment:
We have these changes merged..can switch back to the cloud-bulldozer/e2e-benchmarking

Reply:
yep forgot to move that back, good catch